staging/src/k8s.io/apiserver/pkg/endpoints/discovery/legacy.go

```diff
@@ -32,23 +32,21 @@ import (
 // legacyRootAPIHandler creates a webservice serving api group discovery.
 type legacyRootAPIHandler struct {
 	// addresses is used to build cluster IPs for discovery.
-	addresses   Addresses
-	apiPrefix   string
-	serializer  runtime.NegotiatedSerializer
-	apiVersions []string
```

Comment:
/lgtm
Ideally we should have a comment here explaining that v1 is the only allowed version for legacy APIs. 
I think it would help people working in this code and help us as reviewers not accidentally let a change here through.